README.md

```diff
@@ -1,2 +1,99 @@
 # open-sir
-Python SIR-x model implementation
+
+Open-SIR is an Open Source Python project for modelling pandemics and infectious diseases using Compartmental Models, such as the widely used [Susceptible-Infected-Removed (SIR) model](http://rocs.hu-berlin.de/corona/docs/forecast/model/#classic-sir-dynamics). 
+
+The current stage of the software is *Alpha*.
+
+## Features
+- Model the dynamics of infectious diseases
+- Parameter fitting
+- Calculation of confidence intervals
+- CLI for interfacing with non Python environments (Bash, Node.JS, Matlab, etc).
+
+So far, Open-SIR provides an implementation of the SIR model and the recently new SIR based model SIR-X, developed by the [Robert Koch Institut](http://rocs.hu-berlin.de/corona/docs/forecast/model/#sir-x-dynamics-outbreaks-with-temporally-increasing-interventions).
+
+## Getting Started
+
+Open-SIR uses [Pipenv](https://pipenv.pypa.io/en/latest/) to automatically create a virtual environment and manage python packages. The python packages required by Open-SIR are listed in the [Pipfile](Pipfile).
+
+### Dependencies
```

Comment:
I am not sure about this section. It is a list which could easily get out of date every time someone adds a new dependency to the `Pipfile` and forgets to add it here.

I think it is important to say that you need `Python 3.7` and `Pipenv`, but about the rest of the libraries I am not so sure.

Reply:
I agree with this. Considering it's just a developer environment, both dependencies should suffice.

Reply:
But what if somebody wants to use open-sir before we make it a package?

Reply:
But it can still use it with "pipenv"

Reply:
Yes. Knowing that it uses `Pipenv` for dependency management should be enough. At no moment should that dev manually add the dependencies. All is automatically sorted with `pipenv install`.

Reply:
OK. Implemented